config/kubernetes/sig-release/teams.yaml

```diff
@@ -202,6 +209,9 @@ teams:
     - idealhack # Patch Release Team
     - justaugustus # Branch Manager
     - k8s-release-robot
+    # TODO(justaugustus): Temporary assignment to shadow Branch Managers
+    #                     Remove after 11/5/19.
```

Comment:
correct me if I'm wrong: it seems this is only needed for branch fast-forwarding?